repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/http/media/MimeType.java

```diff
@@ -24,6 +24,8 @@
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
 public enum MimeType {
 
+    // TODO: This list isn't even close to being exhaustive.
+    //       An enum is probably not the right structure for this data since it cannot represent unlisted MIME types.
```

Comment:
I really don't like that this enum even exists, but replacing it proved to be a fair bit of work that is out of scope for this story. I left this comment to express my dislike of this code -- I'd rather use something like JAX-RS' `MediaType`.